### PR TITLE
Validations and spacing to withdrawal modal

### PR DESCRIPTION
**Pull request by constanzauanini**
Screen Shot 2022-09-23 at 14 08 44

Screen Shot 2022-09-23 at 14 22 01

### CODE REVIEW

File: src/components/accession2/withdraw/WithdrawModal.tsx [hunk 1]
```diff
@@ -162,6 +178,32 @@ export default function WithdrawDialog(props: WithdrawDialogProps): JSX.Element
...
+        setIndividualError('withdrawnQuantity', strings.INVALID_VALUE);
+        return false;
+      } else {
+        if (Number(record.withdrawnQuantity.quantity) <= 0) {
+          setIndividualError('withdrawnQuantity', strings.INVALID_VALUE);
+          return false;
+        }
+      }
+    } else {
+      setIndividualError('withdrawnQuantity', strings.REQUIRED_FIELD);
```
Comment (karthikbtf) [inline src/components/accession2/withdraw/WithdrawModal.tsx:200]: if quantity is 0 we will end up here? if so, maybe we need a different message for quantity=0?

**Review by karthikbtf — COMMENTED** [review]
should there be space between the unit dropdown and the amount field?

File: src/components/accession2/withdraw/WithdrawModal.tsx [hunk 2]
```diff
@@ -162,6 +178,32 @@ export default function WithdrawDialog(props: WithdrawDialogProps): JSX.Element
...
+      return false;
+    }
+    setIndividualError('', strings.REQUIRED_FIELD);
```
[the remark this answers is not in the document]
Comment (constanzauanini) [reply src/components/accession2/withdraw/WithdrawModal.tsx:203]: Ups, correct. Thanks

File: src/components/accession2/withdraw/WithdrawModal.tsx (hunk 1 above)
Comment (constanzauanini) [reply src/components/accession2/withdraw/WithdrawModal.tsx:200]: 0 is '0' as string, so you end up in the invalid value message

Comment (karthikbtf) [reply src/components/accession2/withdraw/WithdrawModal.tsx:200]: ah yes, forgot this is a string input

**Review by karthikbtf — APPROVED** [review]
Thanks 👍

**Pull request merged**